autoload/swagger_previewer.vim

```diff
@@ -1,15 +1,18 @@
+let s:vimSwaggerPreviewDir = expand('<sfile>:p:h:h') . "/script.sh"
 
 function! swagger_previewer#Generate(pluginPath)
   let tmpDir = "/tmp/vim-swagger-preview/"
   call system("mkdir " . tmpDir)
-  if exists( "g:swagger_preview_verbose_log" ) 
-    execute "!" . "swagger_preview " . bufname("%")
+
+  let s:cmd = "!'" . s:vimSwaggerPreviewDir . "' '" . bufname("%") . "'"
+
+  silent execute "!echo Please wait..."
+
+  if exists( "g:swagger_preview_verbose_log" )
+    execute s:cmd
   else
-    if has('nvim')
```

Comment:
I haven't been using vim for a while,
does it mean there is no need to differentiate neovim and vim?
@lirik90

Reply:
I'm a beginner in vimscript, and I can't say exactly how long ago this command appeared, but in my ubuntu, vim installed from a regular repository handles this command correctly. In addition, I have seen on stackoverflow the use of this command, in simple vim, in messages from 2016. So I guess there is no need to differentiate neovim and vim. =)